projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -437,7 +405,9 @@ async function go(options) {
 	const remote = await getRemote(options);
 	let from = await getPreviousReleasedVersion(to, tagPattern);
 	const date = await getDate(to);
-	let contents = await generate({
+
+	/* eslint-disable-next-line prefer-const */
```

Comment:
Sigh... one reason why I don't like "abusing" tuples. We want one of these (`contents`) to be reassignable but not the other (`breaking`) so ESLint will complain unless we suppress it.